base/libc.jl

```diff
@@ -96,114 +98,38 @@ else
     error("systemsleep undefined for this OS")
 end
 
-immutable TimeVal
-   sec::Int64
-   usec::Int64
-end
-
-function TimeVal()
-    tv = Ref{TimeVal}()
-    status = ccall(:jl_gettimeofday, Cint, (Ref{TimeVal},), tv)
-    status != 0 && error("unable to determine current time: ", status)
-    return tv[]
-end
-
 """
-    TmStruct([seconds])
```

Comment:
TmStruct is only needed, as far as I can tell, for strptime and strftime. As Jeff mentioned in the last pull request, these are barely ever used. I guess I could split into two pull requests?